docs/usage_guide.rst

```diff
@@ -856,19 +856,19 @@ For every ethereum address it is possible to trigger the process of detecting to
    :align: center
 
 
-Ethereum Transactions
+EVM Transactions
 =====================
 
-rotki is capable of getting and decoding (understand what is happening) your ethereum transactions. When you visit the ``Ethereum Transactions`` section the process to obtain all the information will start. You will be able to check the status in an informative breakdown per address.
+rotki is capable of getting and decoding (understand what is happening) your EVM transactions. When you visit the ``EVM Transactions`` section the process to obtain all the information will start. You will be able to check the status in an informative breakdown per address. Free users are limited to display 100 transactions.
```

Comment:
```suggestion
rotki is capable of getting and decoding (understand what is happening) your EVM transactions. When you visit the ``EVM Transactions`` section the process to obtain all the information will start. You will be able to check the status in an informative breakdown per address. Free users are limited to the first/last 100 transactions.
```

Is it first or last 100?

Reply:
it returns the last 100 tx order by timestamp from newest to oldest